### PR TITLE
Adapt to changes of stdlib run function and CalledProcessError exception

### PR DESCRIPTION
**Pull request by vojtechsokol**
Refactoring after changes of leapp framework in stdlib run function and CalledProcessError exception in oamg/leapp#480.

### CODE REVIEW

**Comment by pirat89** [comment]
```
00:38:08 00:38:08 Process Process-14:
00:38:08 00:38:08 Traceback (most recent call last):
00:38:08 00:38:08   File "/usr/lib64/python2.7/multiprocessing/process.py", line 258, in _bootstrap
00:38:08 00:38:08     self.run()
00:38:08 00:38:08   File "/usr/lib64/python2.7/multiprocessing/process.py", line 114, in run
00:38:08 00:38:08     self._target(*self._args, **self._kwargs)
00:38:08 00:38:08   File "/tmp/repo/tut/lib/python2.7/site-packages/leapp/repository/actor_definition.py", line 26, in inspect_actor
00:38:08 00:38:08     definition.load()
00:38:08 00:38:08   File "/tmp/repo/tut/lib/python2.7/site-packages/leapp/repository/actor_definition.py", line 135, in load
00:38:08 00:38:08     with self.injected_context():
00:38:08 00:38:08   File "/usr/lib64/python2.7/contextlib.py", line 17, in __enter__
00:38:08 00:38:08     return self.gen.next()
00:38:08 00:38:08   File "/tmp/repo/tut/lib/python2.7/site-packages/leapp/repository/actor_definition.py", line 249, in injected_context
00:38:08 00:38:08     map(lambda x: os.path.join(self._repo_dir, self.directory, x), self.libraries))
00:38:08 00:38:08   File "/tmp/repo/tut/lib/python2.7/site-packages/leapp/repository/loader.py", line 19, in library_loader
00:38:08 00:38:08     imported = importer.find_module(name).load_module(name)
00:38:08 00:38:08   File "/usr/lib64/python2.7/pkgutil.py", line 246, in load_module
00:38:08 00:38:08     mod = imp.load_module(fullname, self.file, self.filename, self.etc)
00:38:08 00:38:08   File "/tmp/repo/repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py", line 9, in <module>
00:38:08 00:38:08     from leapp.exceptions import CalledProcessError
00:38:08 00:38:08 ImportError: cannot import name CalledProcessError
00:38:08 00:38:08 ERROR:leapp.repository.system_upgrade_el7toel8:Process inspecting actor in actors/prepareupgradetransaction failed with 1
00:38:08 00:38:08 Traceback (most recent call last):
00:38:08 00:38:08   File "utils/run_pytest.py", line 106, in <module>
00:38:08 00:38:08     repos.load()
00:38:08 00:38:08   File "/tmp/repo/tut/lib/python2.7/site-packages/leapp/repository/manager.py", line 96, in load
00:38:08 00:38:08     repo.load(resolve=False, stage=_LoadStage.ACTORS)
00:38:08 00:38:08   File "/tmp/repo/tut/lib/python2.7/site-packages/leapp/repository/__init__.py", line 153, in load
00:38:08 00:38:08     actor.discover()
00:38:08 00:38:08   File "/tmp/repo/tut/lib/python2.7/site-packages/leapp/repository/actor_definition.py", line 156, in discover
00:38:08 00:38:08     raise ActorInspectionFailedError('Inspection of actor in {path} failed'.format(path=self.directory))
00:38:08 00:38:08 leapp.exceptions.ActorInspectionFailedError: Inspection of actor in actors/prepareupgradetransaction failed
00:38:08 00:38:08 make: *** [test] Error 1
00:38:08 00:38:08 + touch /tmp/failed
```

File: repos/system_upgrade/el7toel8/actors/networkmanagerupdateclientid/actor.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
 from leapp.actors import Actor
-from leapp.libraries.stdlib import CalledProcessError, run
+from leapp.exceptions import CalledProcessError
```
Comment (jmikovic) [inline repos/system_upgrade/el7toel8/actors/networkmanagerupdateclientid/actor.py]:
```suggestion
from leapp.libraries.stdlib import CalledProcessError
```

**Comment by vojtechsokol** [comment]
Yes, this is expected, because right now our CI cannot handle the situation, when there are chenges both in leapp framework and in actors.

**Comment by pirat89** [comment]
Aah, right :)

File: repos/system_upgrade/el7toel8/actors/networkmanagerupdateclientid/actor.py [hunk 2]
```diff
@@ -27,7 +28,7 @@ def process(self):
 
             try:
                 r = run(['nm-update-client-ids.py'], split=True)['stdout']
-            except (OSError, CalledProcessError) as e:
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/networkmanagerupdateclientid/actor.py]: The `OSError` exception can occur as well currently. So this doesn't seem like correct change now.

Comment (bocekm) [reply repos/system_upgrade/el7toel8/actors/networkmanagerupdateclientid/actor.py]: @vojtechsokol, will you be addressing this?

**Comment by vinzenz** [comment]
Closing since  oamg/leapp#480 was closed

**Pull request closed without merging**